modules/virt-importing-vm-cli.adoc

```diff
@@ -130,47 +98,52 @@ spec:
             target:
-              name: <target-network> <8>
+              name: <target_network> <8>
             type: pod
         storageMappings: <9>
           - source:
-              name: <source-storage-domain> <10>
+              name: <source_storage_domain> <10>
             target:
-              name: <target-storage-class> <11>
+              name: <target_storage_class> <11>
+            accessMode: <volume_access_mode> <12>
         diskMappings:
           - source:
-              id: <source-vm-disk-id> <12>
+              id: <source_vm_disk_id> <13>
             target:
-              name: <target-storage-class> <13>
+              name: <target_storage_class> <14>
 EOF
 ----
 <1> If you create a `ResourceMapping` CR, uncomment the `resourceMapping` section.
 <2> Specify the target VM name.
-<3> Specify the source VM ID, for example, `80554327-0569-496b-bdeb-fcbbf52b827b`. You can obtain the VM ID by entering `\https://<RHV_Manager_FQDN>/ovirt-engine/api/vms/` in a web browser on the Manager machine to list all VMs. Locate the VM you want to import and its corresponding VM ID. You do not need to specify a VM name or cluster name.
+<3> Specify the source VM ID, for example, `80554327-0569-496b-bdeb-fcbbf52b827b`. You can obtain the VM ID by entering `\https://www.example.com/ovirt-engine/api/vms/` in a web browser on the Manager machine to list all VMs. Locate the VM you want to import and its corresponding VM ID. You do not need to specify a VM name or cluster name.
 <4> If you specify the source VM name, you must also specify the source cluster. Do not specify the source VM ID.
 <5> If you specify the source cluster, you must also specify the source VM name. Do not specify the source VM ID.
 <6> If you create a `ResourceMapping` CR, comment out the `mappings` section.
 <7> Specify the logical network and vNIC profile of the source VM.
 <8> Specify the {VirtProductName} network.
-<9> If `storageMappings` are specified in both the `ResourceMapping` and the `VirtualMachineImport` CRs, the `VirtualMachineImport` CR takes precedence.
+<9> If storage mappings are specified in both the `ResourceMapping` and the `VirtualMachineImport` CRs, the `VirtualMachineImport` CR takes precedence.
 <10> Specify the source storage domain.
 <11> Specify the target storage class.
-<12> Specify the source VM disk ID, for example, `8181ecc1-5db8-4193-9c92-3ddab3be7b05`. You can obtain the disk ID by entering `\https://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>` in a web browser on the Manager machine and reviewing the VM details.
-<13> Specify the target storage class.
+<12> Specify `ReadWriteOnce`, `ReadWriteMany`, or `ReadOnlyMany`. If no access mode is specified, {virt} determines the correct volume access mode based on the VM *Host* -> *Migration mode* setting:
+* *Allow manual and automatic migration*: `ReadWriteMany`
+* *Do not allow migration*: `ReadWriteOnce`
+* *Allow manual migration only*: `ReadOnlyMany`??
```

Comment:
@pkliczewski  If the VM's migration mode is "Allow manual migration only", what is the volume access mode?

Reply:
In order to enable vm migrations we need `ReadWriteMany`. VM migration is not possible when `ReadWriteOnce`.

Reply:
@pkliczewski  This is about VM import, not VM migration. 

If the user imports a RHV VM with migration mode "Allow manual migration only" and **does not choose a volume access mode**, what access mode will oVirt choose?

Reply:
When access mode is not overridden by the user it will be derived from specific disk setting in RHV.

Reply:
@pkliczewski 
What is the relationship between the RHV disk settings and the access mode **if the access mode is not specified by the user**?

Reply:
RHV disk and PVC for which we set access mode represent vm disk.

Reply:
@pkliczewski 
What is the correlation between RHV disk and virt volume access mode? We want to tell the user what will happen, depending on what their VM settings are.